java/code/src/com/suse/manager/webui/controllers/CVEAuditController.java

```diff
@@ -154,9 +154,8 @@ public static Object cveAudit(Request req, Response res, User user) {
             }
         }
         catch (UnknownCVEIdentifierException e) {
-            return json(res, ResultJson.error("The specified CVE number was not" +
-                    " found. This can happen for very old or yet-unknown numbers, please" +
-                    " also check it for possible typing errors."));
+            return json(res, ResultJson.error("According to our records, your systems/images are not affected by " +
```

Comment:
We should extract this to our String Resource files so it could be translated as well.